database/security-rules.json

```diff
@@ -160,7 +182,7 @@
         "$userActions": {
           ".write": "newData.hasChildren(['publicId','action','timestamp'])",
           "publicId": {
-            ".validate": "newData.isString() && root.child('auth/publicIds').child(newData.val()).val() === auth.uid"
+            ".validate": " true || newData.isString() && root.child('auth/publicIds').child(newData.val()).val() === auth.uid"
```

Comment:
@dinoboff This was for the purpose of team activities. In an team, only the leader submits an answer, but we want to log all users as having collaboratively submitted something. Hence this security rule of blocking writes for user actions not containing one's own publicID was an issue. We will see what work arounds we can apply, but tentatively we shall leave it as it is